bindings/rust/generate/_collect_symbols_build.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
What is the plan for windows?

Reply:
I'd like to add Windows support to this crate as well, but this first iteration doesn't provide that. We can prioritize that next if needed.

Reply:
The scripts in the "generate" directory currently must execute on a Mac that has access to our Linux (x64/aarch) Docker images.  A Mac can utilize its local (Mac-specific) dev tools as well as run scripts on the docker images to build and test the crate, all from a single host. I'm open to suggestions on better ways to do this. 

To support Windows, just the `_collect_symbols_build.sh` and `_crate_test_build.sh` scripts would either need to rewritten (in Rust?), or we'd need to setup a Windows docker image so it could run these bash scripts.